lib/ain-rs-exports/src/evm.rs

```diff
@@ -1136,11 +1136,20 @@ pub fn evm_try_get_tx_sender_info_from_raw_tx(
         return cross_boundary_error_return(result, "nonce value overflow");
     };
 
+    let Ok(prepay_fee) = SERVICES.evm.core.calculate_prepay_gas_fee(&signed_tx) else {
```

Comment:
Can switch fee module to public `pub mod fee;`
and call `calculate_prepay_gas_fee` directly
```
    let Ok(prepay_fee) = calculate_prepay_gas_fee(&signed_tx) else {
        return cross_boundary_error_return(result, "failed to get fee from transaction");
    };
```